source/use-cases/sensitive-data-encryption.txt

```diff
@@ -57,12 +57,122 @@ cubilia Curae; Nunc non interdum purus, ultricies laoreet tortor.
 B. Define a JSON Schema
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-Proin non mi a felis luctus vulputate nec pharetra odio. Ut pretium
-scelerisque nulla, sed euismod nisi pellentesque sit amet. Proin mollis
-mauris eu libero ullamcorper rutrum. Pellentesque vitae enim sed magna
-accumsan maximus et quis mi. Ut vel laoreet turpis. Vivamus vitae odio
-sed arcu finibus pretium vel maximus nisi. Nullam gravida consequat
-porta.
+MongoDB uses JSON Schema to enforce particular datatypes for specific fields during insert and update operations. MongoDB CSFLE requires JSON Schema to identify fields to automatically encrypt and decrypt, the data encryption key id, and the type of encryption algorithm to use:
```

Comment:
Thanks for updating this sentence. I am not sure exactly what I had recommended, but I think to make it work with the list on lines 62-64, you could update the second sentence to something like this:
```
MongoDB CSFLE requires the following properties in the JSON Schema to identify the fields that should be automatically encrypted and decrypted:
```

Reply:
Fixed